src/sst/core/cfgoutput/pythonConfigOutput.h

```diff
@@ -10,46 +10,47 @@
 // distribution.
 //
 
-#ifndef _H_SST_CORE_CONFIG_OUTPUT_PYTHON
-#define _H_SST_CORE_CONFIG_OUTPUT_PYTHON
+#ifndef SST_CORE_PYTHON_CONFIG_OUTPUT_H
+#define SST_CORE_PYTHON_CONFIG_OUTPUT_H
 
 #include "sst/core/configGraph.h"
 #include "sst/core/configGraphOutput.h"
+
 #include <map>
 
 namespace SST {
 namespace Core {
 
-class PythonConfigGraphOutput : public ConfigGraphOutput {
+class PythonConfigGraphOutput : public ConfigGraphOutput
+{
 public:
     PythonConfigGraphOutput(const char* path);
 
-    virtual void generate(const Config* cfg,
-            ConfigGraph* graph) override;
+    virtual void generate(const Config* cfg, ConfigGraph* graph) override;
 
 protected:
     ConfigGraph* getGraph() { return graph; }
-    void generateParams( const Params &params );
-    void generateCommonComponent( const char* objName, const ConfigComponent* comp);
-    void generateSubComponent( const char* owner, const ConfigComponent* comp );
-    void generateComponent( const ConfigComponent* comp );
-    void generateStatGroup(const ConfigGraph* graph, const ConfigStatGroup &grp);
+    void         generateParams(const Params& params);
```

Comment:
This looks kind of wierd, especially since the variables then don't line up. Is this an actual standard we are going to stick to?

Reply:
It's set to line up lines that are not separated by spaces after their types.  So, since one of the functions returns a ConfigGraph*, all the ones that return void are aligned.  It's not my favorite, but it looks okay in the source file.  The flag is really turned on for variables, but the same flag seems to cover functions as well.